modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/parse/MBObjectParser.java

```diff
@@ -885,4 +1113,65 @@ public Expression bool(JSONObject json, String tag, boolean fallback)
                     + obj.getClass().getSimpleName());
         }
     }
+    
+
+    /**
+     * Maps a json value at 'tag' in the provided JSONObject to a {@link Displacement}.
+     * 
+     * @param json The JSONObject in which to look up a displacement value
+     * @param tag The tag in the JSONObject
+     * @param fallback The fallback displacement, if no value is found at that tag.
+     * @return A displacement from the json
+     */
+    public Displacement displacement(JSONObject json, String tag, Displacement fallback) {
```

Comment:
@vickdw supplied [GEOT-5699 SLDTransformer fails to write the Displacement property of a PolygonSymbolizer](https://osgeo-org.atlassian.net/browse/GEOT-5699)

@aaime has (GEOT-5497 Handle fill displacement in map rendering)[https://osgeo-org.atlassian.net/browse/GEOT-5497]

@vickdw ignoring writing out the SLD (because who cares) I though you found that displacement was not used by the renderer?